packages/palette/src/palette.story.tsx

```diff
@@ -54,6 +54,13 @@ function ColorBlock({ color, name }: ColorBlockProps) {
 
 export default {
   title: 'Packages/Palette',
+  argTypes: {
```

Comment:
Is it possible to do something like this on a global level? I feel like we'll want to remove `className` and other props from most other components

Reply:
It is possible - I actually set it to `{ description: "....", control: { type: 'text' }` globally though because I thought we might want it in the future? What I did here in `palette` completely removes `className` from the table, which I don't know if we want globally since that suggests that the className prop doesn't exist on those components.

We can also define it as  `{ description: "....", control: { type: 'none' }`. This would render the row in the prop table along with its description, but without the control. Maybe this is what we want instead. I can make this change some time in the future.